opam

```diff
@@ -32,8 +32,14 @@ build-test: [
   [make]
   ["./test.sh"]
 ]
-install: [make "install"]
-remove:  ["ocamlfind" "remove" "cstruct"]
+install: [
+  [make "install"]
+  [make "js-install"]
```

Comment:
I thought it wasn't worth the bother: nothing needs to be built, so it seems artificial to rebuild cstruct (and all packages dependent on it) if we install js_of_ocaml.

Ideally in future ocamlfind and oasis would know about js stubs as well as C stubs and this would all be automatic.